3-deployment-targets-adapters-and-stacks/bee-rich/README.md

```diff
@@ -1,10 +1,10 @@
 # 3. Deployment Targets, Adapters, and Stacks
 
-In this chapter, we will introduce you to BeeRich, the demo application used throughout this book. BeeRich is a simple personal finance app that allows you to track expenses and your income. Each chapter will add more features to BeeRich and focus on a specific topic.
+In this chapter, we will introduce you to BeeRich, the demo application used throughout this book. BeeRich is a simple personal finance app that allows you to track your expenses and income. Each chapter will add more features to BeeRich and focus on a specific topic.
 
 ## Getting started
 
-**Welcome!** BeeRich is a dashboard-like application that mimics both personal and enterprise use cases. BeeRich is a personal finance management application that helps you stay on top of your bee - pardon me - bookkeeping. Well, at least that’s the goal. There is nothing much there yet. In every chapter, we will add more code to this application.
+**Welcome!** BeeRich is a dashboard-like application that mimics both personal and enterprise use cases. BeeRich is a personal finance management application that helps you stay on top of your beekeep - pardon me - bookkeeping. Well, at least that’s the goal. There is nothing much there yet. In every chapter, we will add more code to this application.
```

Comment:
You don't have to make this change of course, I only added it because I had to read the joke twice to understand it. Could have been my lack of a sense of humor 😉